02-household-queries/chainlit-household-bot.py

```diff
@@ -34,10 +55,17 @@ async def init_chat():
         [
             Select(
                 id="model",
-                label="LM Model",
-                values=OLLAMA_LLMS + OTHER_LLMS,
+                label="LLM Model",
+                values=OLLAMA_LLMS + GOOGLE_LLMS + GPT4ALL_LLMS,
+                initial_index=0,
+            ),
+            Select(
+                id="embedding",
+                label="Embeddings",
+                values= GOOGLE_EMBEDDINGS + OPEN_SOURCE_EMBEDDINGS + HUGGING_FACE_EMBEDDINGS,
                 initial_index=0,
             ),
+            Switch(id="use_vector_db", label="Use vector db sources", initial=False),
```

Comment:
I suggest setting the `initial` values to be based on environment variables so you don't have to change the settings with each restart.